modules/claims_api/app/controllers/claims_api/v0/forms/disability_compensation_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_dependency 'claims_api/application_controller'
+require 'jsonapi/parser'
+
+module ClaimsApi
+  module V0
+    module Forms
+      class DisabilityCompensationController < ApplicationController
+        skip_before_action(:authenticate)
+        # before_action :validate_json_api_payload
+
+        def form_526
+          auto_claim = ClaimsApi::AutoEstablishedClaim.create(
+            status: ClaimsApi::AutoEstablishedClaim::PENDING,
+            auth_headers: auth_headers,
+            form_data: form_attributes
+          )
+          auto_claim.form.to_internal
+
+          ClaimsApi::ClaimEstablisher.perform_async(auto_claim.id)
+
+          render json: auto_claim, serializer: ClaimsApi::AutoEstablishedClaimSerializer
+        rescue RuntimeError => e
+          render json: { errors: e.message }, status: 422
+        end
```

Comment:
I'm fine with this unconventional action name but maybe it could include a verb to indicate (to someone looking through logs, metrics, etc) that this is accepting a new form rather than displaying the form? `create_form_526` or `submit_form_526` perhaps?

Reply:
Will update to submit form 526 to better signify the action it's taking

Reply:
resolved in 499e686